runtime-decompiler/src/main/java/org/jrd/frontend/frame/nestedJars/NestedJarsView.java

```diff
@@ -0,0 +1,181 @@
+package org.jrd.frontend.frame.nestedJars;
+
+import org.jrd.backend.data.ArchiveManagerOptions;
+import org.jrd.frontend.frame.main.MainFrameView;
+
+import javax.swing.*;
+import javax.swing.border.EtchedBorder;
+import java.awt.*;
+import java.awt.event.ItemEvent;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class NestedJarsView extends JDialog {
+	private JPanel mainPanel;
+	private NestedPanel nestedPanel;
+	private JPanel okCancelPanel;
+	private JButton okButton;
+	private JButton cancelButton;
+
+	public static class NestedPanel extends JPanel {
+
+		public JCheckBox useDefaults;
+		public JTextField textField;
+		public JButton addButton;
+		public JButton removeButton;
+		public JList list;
+		public JScrollPane scrollPane;
+		DefaultListModel dList;
+
+		NestedPanel() {
+			textField = new JTextField();
+
+			dList = new DefaultListModel();
+			list = new JList(dList);
+			list.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
+			list.setLayoutOrientation(JList.VERTICAL);
+			list.setVisibleRowCount(-1);
+
+			scrollPane = new JScrollPane(list);
+			scrollPane.setPreferredSize(new Dimension(0, 200));
+
+			addButton = new JButton("ADD");
+			addButton.addActionListener(actionEvent -> {
+				dList.addElement("." + textField.getText());
+				textField.setText("");
+			});
+
+			removeButton = new JButton("REMOVE");
+			removeButton.addActionListener(actionEvent -> {
+				int index = list.getSelectedIndex();
+				dList.removeElementAt(index);
+			});
+
+			useDefaults = new JCheckBox("Use default extensions");
```

Comment:
pls use as tooltip, ot better as something waht can be copypasted, to show current list of default extensions

Reply:
Tooltip added. It could also be added into the text, let me know if that's something you'd like.